app/assets/javascripts/organisation-autocomplete.js

```diff
@@ -5,16 +5,118 @@ window.GOVUKAdmin.Modules = window.GOVUKAdmin.Modules || {};
 
   Modules.OrganisationAutocomplete = function () {
     this.start = function ($element) {
-      accessibleAutocomplete.enhanceSelectElement({
-        selectElement: $element.find('#organisations')[0],
-        showAllValues: true,
-        autoselect: false,
-        dropdownArrow: dropdownArrow
-      });
-    };
 
-    function dropdownArrow() {
-      return '<span class="caret"></span>';
-    }
+      var autocompleteTemplateHtml = createAutocompleteTemplate();
+
+      $element
+        .append(addOrganisationButton())
+        .find('[selected]')
+        .each(function (index, option) {
+          if (index > 0) {
+            $(autocompleteTemplateHtml)
+              .insertAfter(lastSelectWrapper())
+              .find('option[value="' + $(option).attr('value') + '"]')
+              .attr('selected', 'selected')
+          }
+        });
+
+      enableAutocomplete();
+
+      function createAutocompleteTemplate() {
```

Comment:
Have you considered embedding the template within the HTML, hide it via CSS and clone it with `JQuery`?
I find this approach more readable and maintainable.

Reply:
I've done this sort of approach in the past, and it works pretty well. It sort of just moves the mess from JS to Ruby/CSS, but I don't mind either way.